composition-js/src/types.ts

```diff
@@ -0,0 +1,8 @@
+import {
```

Comment:
Nite: I'm note sure why this file is called `types`. What's the reasoning?

Reply:
I guess I was thinking that it could be a general place for putting types that might otherwise resolve in a circular dependency, but I'll name it to be more specfic.